src/control/lib/hardware/sysfs/provider_test.go

```diff
@@ -199,6 +255,35 @@ func setupVirtualIB(t *testing.T, root, virtDev, parent string) {
 	}
 }
 
+func setupNetvscDev(t *testing.T, root, devName, backingDevName string) {
```

Comment:
if we have added a special case to the discovery routine, why do we need to add symbolic links to the device? can't we access the device via the discovered path?

Reply:
Not sure to understand your question?
This function is for creating a mock NetVSC NIC as it is done with `setupPCIDev()` function for standard PCI device.

Reply:
ah, apologies I didn't see this was a test file, ignore this comment